tdrs-backend/tdpservice/core/admin.py

```diff
@@ -33,22 +33,6 @@ class LogEntryAdmin(admin.ModelAdmin):
         'action_flag',
     ]
 
-    def has_add_permission(self, request):
```

Comment:
These methods exist as overrides to the default model permissions, which we now leverage so this is unnecessary.